common/src/main/resources/assets/figura/lang/en_us.json

```
@@ -1304,22 +1305,22 @@
     "figura.docs.text_task.set_alignment": "Sets this Text alignment\nCan be either \"LEFT\", \"RIGHT\" or \"CENTER\"\nDefault \"LEFT\"",
     "figura.docs.text_task.has_shadow": "Checks if this task text has shadow",
     "figura.docs.text_task.set_shadow": "Toggles if the Text should render with a drop shadow\nNot compatible with \"Outline\" mode",
-    "figura.docs.text_task.has_outline": "Checks if this task text has outline",
-    "figura.docs.text_task.set_outline": "Toggles if the Text should render with a outline\nAlways renders at full bright\nNot compatible with \"Shadow\" and \"Emissive\" modes",
+    "figura.docs.text_task.has_outline": "Checks if this task text has an outline",
+    "figura.docs.text_task.set_outline": "Toggles, if the text should render with an outline\nAlways, renders at full bright\nNot compatible with \"Shadow\" and \"Emissive\" modes",
     "figura.docs.text_task.get_outline_color": "Gets this tasks text outline color",
     "figura.docs.text_task.set_outline_color": "Sets the outline color this Text should render\nOnly compatible with \"Outline\" mode",
     "figura.docs.text_task.get_width": "Gets this Text max width\nDefault 0",
```

Changed the last line to match the next suggestion.
Made "text" lowercase because it doesn't really specify the TextTask itself.
```suggestion
    "figura.docs.text_task.get_width": "Gets this text's max width\nDefaults to 0",
```